x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/BooleanArrayBlock.java

```diff
@@ -50,6 +50,23 @@ private BooleanArrayBlock(
     ) {
         super(positionCount, firstValueIndexes, nulls, mvOrdering, blockFactory);
         this.vector = vector;
+        checkInvariants();
```

Comment:
I think we should name this method `assertInvariants` and return a boolean, then call `assert checkInvariants()` instead.